### PR TITLE
feat: run geometry checks

### PR DESCRIPTION
**Pull request by wdconinc**
### Briefly, what does this PR introduce?
DD4hep has the checkGeometry tool which can be used to produce histograms of time per track spent per volume. This workflow produces these histograms and log output for both fast and full geometry.

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [X] New feature (issue #__)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [ ] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?
No.

### Does this PR change default behavior?
No.

### CODE REVIEW

File: .github/workflows/linux-eic-shell.yml [hunk 1]
```diff
@@ -233,6 +233,60 @@ jobs:
...
+        name: build-fast-eic-shell
+        path: install/
+    - uses: cvmfs-contrib/github-action-cvmfs@v3
+    - uses: eic/run-cvmfs-osg-eic-shell@main
+      with:
+        platform-release: "jug_xl:nightly"
+        setup: install/setup.sh
+        run: |
+          mkdir -p doc
+          checkGeometry -f true -c ${DETECTOR_PATH}/${DETECTOR_CONFIG}.xml | tee geometry_check_fast.out
```
Comment (veprbl) [inline .github/workflows/linux-eic-shell.yml:253]: If this is "fast", then it should be `-f false` here?

Comment (wdconinc) [reply .github/workflows/linux-eic-shell.yml:253]: Yeah, probably. Originally the goal was to have this work on the fast geometry, no fibers and such, but it doesn't run faster than the full geometry. So yeah, we'd probably switch to false.

Comment (veprbl) [reply .github/workflows/linux-eic-shell.yml:253]: Or remove, if it's currently identical.

Comment (wdconinc) [reply .github/workflows/linux-eic-shell.yml]: Removed.

**Review by veprbl — APPROVED** [review]

**Pull request merged**